setup.py

```diff
@@ -2,19 +2,42 @@
 
 from setuptools import setup
 
+import i18n
```

Comment:
@clintonb Do you have thoughts on this? I know there was a comment before on this PR to use the __version__ constant.